cosmwasm/packages/enclave-ffi-types/src/types.rs

```diff
@@ -165,6 +165,21 @@ pub enum NodeAuthResult {
     Panic,
 }
 
+/// This type represents the possible error conditions that can be encountered in the
+/// enclave while authenticating a new node in the network.
+/// cbindgen:prefix-with-name
+#[repr(C)]
+#[derive(Debug, Display, PartialEq, Eq)]
+pub enum HealthCheckResult {
+    Success,
+}
```

Comment:
This enum only has one variant? that's odd

Reply:
One day we might want to make this check other things and return more detailed responses. So it's an enum for easy upgradability